Add mentions about Gradle support in Designer (#352)
vaadin/docs: articles/tools/designer/getting-started/build-your-main-view.asciidoc

Vaadin Designer will open in Google Chrome.

> WARNING: If you are using a <<{articles}/guide/start/gradle#, Vaadin Gradle project>>, make sure you have run the project at least once so the frontend dependencies (i.e. `node_modules` folder) are ready for Designer.

Reviewer: ```suggestion
WARNING: If you are using a <<{articles}/guide/start/gradle#, Vaadin Gradle project>>, make sure you have run the project at least once so the frontend dependencies (in the `node_modules` folder) are ready for Designer.
```
